sirepo/package_data/static/js/sirepo-components.js

```diff
@@ -3600,13 +3600,22 @@ SIREPO.app.directive('emailLogin', function(requestSender, errorService) {
             </div>
         `,
         controller: function($scope) {
+            function showInvalidEmail() {
+                $scope.showWarning = true;
+                $scope.warningText = 'Email address is invalid. Please update and resubmit.';
```

Comment:
I think, in this case, we should say "please use your institutional email, we do not allow ${domain} addresses". This could be generated server side.

Reply:
34cc6ad44